examples/quickstarts/helidon-standalone-quickstart-se/pom.xml

```diff
@@ -179,6 +180,14 @@
                     <artifactId>helidon-maven-plugin</artifactId>
                     <version>${version.plugin.helidon}</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
```

Comment:
How about just defining the property `<maven.site.skip>true</maven.site.skip>`

Reply:
Fixed